Automattic/pocket-casts-ios
# Referrals: Send pass UI

| 📘 Part of: #2083  |  
|:---:|

Fixes # 

This PR implements the UI only for the Send Pass UI for Referrals.

> It still does not implement the background animated gradient on the Cards. That will be done a separate PR.

| 1 | 2 | 3 | 
| - | - | - |
| ![Simulator Screenshot - iPhone 15 Pro - 2024-09-04 at 16 54 06](https://github.com/user-attachments/assets/df0a5800-60e7-4709-be3b-39dcb7bf09d2) | ![Simulator Screenshot - iPad mini (6th generation) - 2024-09-04 at 16 56 39](https://github.com/user-attachments/assets/0b08d12f-86e0-4495-926e-16bb385d8fa1) | ![Simulator Screenshot - iPad mini (6th generation) - 2024-09-04 at 16 56 24](https://github.com/user-attachments/assets/b63f2599-5edd-425c-95cf-da975488716e) | 


## To test

1. Start the app
2. Ensure that you have the Transcripts FF enabled and you logged with a Plus or Patron account
3. Go to Profile View Controller
4. Tap on the gift icon on the top left
5. Check if the new guest pass screen is show
6. Tap on the close button
7. Tap on the share guest pass button
8. Test on the iPad

## Checklist
